db/migrate/20180108144114_drop_mail_templates_table.rb

```diff
@@ -0,0 +1,12 @@
+class DropMailTemplatesTable < ActiveRecord::Migration[5.0]
+  def change
+    drop_table "mail_templates", force: :cascade do |t|
```

Comment:
On a besoin de préciser ? le `drop_table :mail_templates` ne suffit pas ?

Reply:
c'est pour rendre réversible. Si tu penses que ça ne sert à rien j’enlève volontiers

Reply:
Hmm il me semble que la bonne pratique ici pour rollback une migration est d'en créer une nouvelle ensuite. On avait déjà eu le débat je crois. @LeSim tu confirmes ?

Reply:
Heuu ... je me souviens plus ... perso ca me va d'utiliser change lorsque cela permet de coder up and down en mm temps.

Reply:
Disons que je pense qu'on ne se servira jamais de down donc pas besoin d'ecrire un change verbeux, un change minimal convient.

Reply:
Du coup, un

```ruby
  def down
    raise ActiveRecord::IrreversibleMigration
  end
```
ou même pas ?

Reply:
Je dirais meme pas :P